media-sound/qsynth/qsynth-0.4.2-r1.ebuild

```diff
@@ -2,64 +2,51 @@
 # Distributed under the terms of the GNU General Public License v2
 # $Id$
 
-EAPI=5
+EAPI=6
 
-inherit qmake-utils eutils
+inherit eutils flag-o-matic qmake-utils
 
 DESCRIPTION="A Qt application to control FluidSynth"
 HOMEPAGE="http://qsynth.sourceforge.net/"
 SRC_URI="mirror://sourceforge/qsynth/${P}.tar.gz"
 
 LICENSE="GPL-2"
 SLOT="0"
-IUSE="debug jack alsa pulseaudio +qt5"
+IUSE="debug jack alsa pulseaudio"
 KEYWORDS="~amd64 ~ppc ~x86"
 
 RDEPEND="
-	qt5? (
-		dev-qt/qtwidgets:5
-		dev-qt/qtx11extras:5
-		dev-qt/qtcore:5
-	)
-	!qt5? (
-		>=dev-qt/qtcore-4.2:4
-		>=dev-qt/qtgui-4.2:4
-	)
-	>=media-sound/fluidsynth-1.0.7a[jack?,alsa?,pulseaudio?]
+	dev-qt/qtcore:5
+	dev-qt/qtwidgets:5
+	dev-qt/qtx11extras:5
+	media-sound/fluidsynth[jack?,alsa?,pulseaudio?]
 	x11-libs/libX11"
 DEPEND="${RDEPEND}
-	qt5? ( dev-qt/linguist-tools:5 )
+	dev-qt/linguist-tools:5
 "
 
 REQUIRED_USE="|| ( alsa jack pulseaudio )"
 
-src_prepare() {
-	use qt5 && epatch "${FILESDIR}/${PN}-0.4.0-qt5-tagging.patch"
-}
+DOCS=( AUTHORS ChangeLog README TODO TRANSLATORS )
+
+PATCHES=( "${FILESDIR}/${PN}-0.4.0-qt5-tagging.patch" )
 
 src_configure() {
+	append-cxxflags -std=c++11
 	econf \
 		$(use_enable debug) \
-		$(use_enable !qt5 qt4) \
-		--with-qt"$(usex qt5 "5=$(qt5_get_bindir)/.." "4=$(qt4_get_bindir)/..")"
+		--with-qt5=$(qt5_get_bindir)/..
 
-	# Emulate what the Makefile does, so that we can get the correct
-	# compiler used.
-	if use qt5 ; then
-		eqmake5 ${PN}.pro -o ${PN}.mak
-	else
-		eqmake4 ${PN}.pro -o ${PN}.mak
-	fi
+	eqmake5 ${PN}.pro -o ${PN}.mak
 }
 
 src_install () {
+	default
```

Comment:
you sure this is a good idea? rather not call `default`, and call `einstalldocs` after the `emake` call